### ISSUE TITLE
db.create_table() and db.create_tables([,]) have different behavior？

### ISSUE THREAD

**Issue by piperck** [coleifer/peewee#1269]
```python
class Account(BaseModel):

    class Meta:
        db_table = 'account'
        indexes = (
            (('app_id', 'user_id'), True),
        )

    app_id = IntegerField()
    user_id = IntegerField()
    amount = DecimalField(15, 2)
    create_time = DateTimeField(default=datetime.datetime.now)
    update_time = DateTimeField(default=datetime.datetime.now)
```
I try to create my tables.And I found if I use `create_table(Account)` can not make `indexes=(xx)` coming into effect.
I use `show create table account` check my mysql_tables. It shows
```sql
| account | CREATE TABLE `account` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `app_id` int(11) NOT NULL,
  `user_id` int(11) NOT NULL,
  `amount` decimal(15,2) NOT NULL,
  `create_time` datetime NOT NULL,
  `update_time` datetime NOT NULL,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci |
```

But I use `create_tables([Account,])`, It works！
```sql
| account | CREATE TABLE `account` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `app_id` int(11) NOT NULL,
  `user_id` int(11) NOT NULL,
  `amount` decimal(15,2) NOT NULL,
  `create_time` datetime NOT NULL,
  `update_time` datetime NOT NULL,
  PRIMARY KEY (`id`),
  UNIQUE KEY `account_app_id_user_id` (`app_id`,`user_id`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci |
```

Is this a problem or expectable?

**Comment by coleifer** [comment]
These methods are very confusingly named -- apologies for that.

History:

Originally, you would use `Model.create_table()` which would create the table along with the indexes and other constraints. The `Database.create_table()` method was used behind-the-scenes and it **only** creates the table -- not indexes.

Then I realized there is a common situation where you need to create many tables, and create them in the right order so that the dependency graph is created correctly. To accomplish this, I added the `Database.create_tables()` method (table**s**). It accepts a list of models, does a topological sort to obtain a total ordering, then calls `Model.create_table()` on each table in succession, which ensures the indexes are created as well.

So, long story short, `Database.create_table()` should not be used unless you have a very specific reason for using it.

You should use `Model.create_table()` to create a single model's table & indexes, and `Database.create_tables()` when you want to create tables & indexes for more than one model.

**Comment by piperck** [comment]
ok! Thanks for your long story and long story short !!
 It entirely solve my problem!🤣😇
